dist/index.html

```diff
@@ -14,6 +14,43 @@
     <div id="swagger-ui"></div>
     <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
     <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
-    <script src="./swagger-initializer.js" charset="UTF-8"> </script>
+    <script src="./swagger-initializer.js" charset="UTF-8">
+    
+    const oidcConfig = {
+      clientId: "your-client-id",
+      discoveryUrl: "https://your-auth-server/.well-known/openid-configuration",
+      oauth2RedirectUrl: window.location.origin + "/oauth2-redirect.html"
+    };
+
+    window.onload = function () {
+
+      const ui = SwaggerUIBundle({
+  url: "/path/to/openapi.yaml",  // Make sure this points to your OpenAPI spec
+  dom_id: '#swagger-ui',
+  deepLinking: true,
+  presets: [
+    SwaggerUIBundle.presets.apis,
+    SwaggerUIStandalonePreset
+  ],
+  layout: "StandaloneLayout",
+
+  oauth2RedirectUrl: window.location.origin + "/oauth2-redirect.html",  // OAuth2 redirect URL
+
+  // OAuth2 authentication configuration
+  oauth2: {
+    clientId: "YOUR_CLIENT_ID",   // Replace with actual client ID
+    clientSecret: "YOUR_CLIENT_SECRET", // Optional, only needed for some flows
```

Comment:
I would delete this line of code. It's bad practice to expose client secrets on a HTML/JS page that can everybody read. Use a public OIDC client configuration only which doesnt require a client secret.